### ISSUE TITLE
Elbow supination contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7691]
**Preferred term label:**
Elbow supination contracture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Let's name the term "supination contracture of the forearm"

**Comment by pnrobinson** [comment]
added
